NEWSLETTER_README.md

```
@@ -0,0 +1,232 @@
+│   └── goodbye.tsx                  # Abmeldungs-E-Mail
+└── lib/
+    └── newsletter-schema.ts         # Zod Validation Schema
+```
+
+### Environment Variables
+
+Füge zu deiner `.env` hinzu:
+
+```env
+NEXT_PUBLIC_SITE_URL=http://localhost:3000
+STRAPI_API_URL=http://localhost:1337
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)